nex/matchmake-extension/cleanup_search_matchmake_session.go

```diff
@@ -0,0 +1,10 @@
+package nex_matchmake_extension
+
+import (
+	matchmaking_types "github.com/PretendoNetwork/nex-protocols-go/match-making/types"
+)
+
+func CleanupSearchMatchmakeSession(matchmakeSession *matchmaking_types.MatchmakeSession) {
+	// Cleanup VR
+	matchmakeSession.Attributes[1] = 0
+}
```

Comment:
I'm not sure this function should here. Typically functions inside `nex/PROTOCOL` are for NEX methods, but this is a helpder function for the common protocols